hi-ml/src/health_ml/utils/model_util.py

```diff
@@ -0,0 +1,28 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from typing import Iterator
+
+import torch
+from torch.nn.parameter import Parameter
+from torch.optim.rmsprop import RMSprop
+
+from health_ml.deep_learning_config import OptimizerParams, OptimizerType
+
```

Comment:
Similar to the above comment about LR schedulers, should this really be in hi-ml?